pyproject.toml

```diff
@@ -62,6 +62,7 @@ matplotlib = "^3.7.0"
 keras = "^3.0.0"
 tensorflow = { version = "^2.16.1", markers = "sys_platform == 'linux' or sys_platform == 'darwin'"}
 torch = { version = "^2.3.1"}
+#qibolab = { git="https://github.com/qiboteam/qibolab", extras=["emulator"] }
```

Comment:
We need to uncomment this, right?

Reply:
We already discussed a solution with @scarrazza and @stavros11. It will be delivered soon.

However, notice that the solution will consist in limiting the dependencies, getting rid of optional ones as much as possible. This is something that should be considered also in `qiboml` (which starts proliferating, and it will lead to very similar problems).